apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/ReadRowsTest.cs

```diff
@@ -478,11 +478,7 @@ public void ReadRowMissing()
         [Fact]
         public async Task ReadRows_Validate_SingleIteration()
         {
-            var tableName = new TableName(
-                _fixture.InstanceName.ProjectId,
-                _fixture.InstanceName.InstanceId,
-                "ReadRows_Validate_SingleIteration_Table");
-            await _fixture.CreateTable(tableName);
+            var tableName = await _fixture.CreateTable();
```

Comment:
Is this the only test that needs to create a new table? That's surprising.

Reply:
Most tests use the default table created by the fixture and just insert a row with a random row key. But this one creates its own table because it iterates the rows to make sure only one row is present.

Reply:
Right, makes sense.